app/components/x-toggle.js

```diff
@@ -5,8 +5,6 @@ const config = ENV['ember-cli-toggle'] || {};
 
 export default XToggle.extend({
   theme: config.defaultTheme || 'default',
-  offLabel: config.defaultOffLabel || 'Off',
-  onLabel: config.defaultOnLabel || 'On',
```

Comment:
Why not just set it as a property on `addon/components/` ... that's working now I think